specifyweb/frontend/js_src/lib/localization/schema.ts

```diff
@@ -289,4 +292,10 @@ export const schemaText = createDictionary({
     'ru-ru': 'Перейти к началу',
     'uk-ua': 'Перейти вгору',
   },
+  fieldID: {
```

Comment:
All keys must be in strictCamelCase. Instead of `fieldID`, do `fieldId` - these should never be two consecutive capitalized letters.

See https://medium.com/@thomasreggi/enforcing-strict-camelcase-5332c53a7484 for some reasons why.

You can find more rules regarding localization here - https://github.com/specify/specify7/blob/production/specifyweb/frontend/js_src/lib/localization/README.md